Infrastructure/gpg-ipdeny/manifest.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+applications:
+  - name: gpg-ipdeny
+    instances: 1
+    memory: 256M
```

Comment:
We will want to think about this number (and this app generally) during the load test - as all our traffic will be going through it. Not one for this review but for a few weeks' time